assets/stylesheets/_variables.scss

```diff
@@ -37,3 +37,6 @@ $base-font-family: 'JetBrains Mono', system-ui, -apple-system, 'Segoe UI', 'Robo
 $base-font-icon-family: 'Material Icon', system-ui, -apple-system, 'Segoe UI', 'Roboto', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji';
 $base-line-height: 1;
 $base-font-color: map-get($brand-color, 'shark');
+
+$zindex-header: 1000;
+$zindex-alert: 1070;
```

Comment:
Prefer to create an SCSS map for z-index to make it more generic and not specific to UI elements. this can become messy over time. This is what we use in most projects:

```
$zIndex: (
  behind: -1,
  default: 1,
  high: 5,
  higher: 100,
  highest: 1100,
  sky: 10000
);
```